libass/ass_render.c

```diff
@@ -44,6 +44,22 @@
 #define BLUR_PRECISION (1.0 / 256)  // blur error as fraction of full input range
 
 
+static int text_info_init(TextInfo* text_info)
```

Comment:
Don’t we use `bool` for this kind of returns now?

Reply:
Sure, done.